.circleci/config.yml

```diff
@@ -3,17 +3,23 @@
 # Check https://circleci.com/docs/2.0/language-ruby/ for more details
 #
 version: 2.1
+parameters:
+  ruby-version:
+    type: string
+    default: "3.0.3"
+  postgres-version:
+    type: string
+    default: "11.13"
 executors:
   docker-publisher:
     environment:
       IMAGE_NAME: suldlss/suri-rails
     docker:
-    - image: circleci/buildpack-deps:stretch
+    - image: cimg/buildpack-deps:stretch
```

Comment:
I don't think this is an existing image.

Reply:
is this old boilerplate?  Why didn't the build fail?

Reply:
Because this step only runs after you merge to master

Reply:
fixed;  found the right image (`cimg/base`), I think.